ocl/oclapi/filters.py

```diff
@@ -138,4 +144,27 @@ def _filter_queryset(self, request, queryset, view, sqs):
 
 class HaystackSearchFilter(BaseHaystackSearchFilter):
     def filter_queryset(self, request, queryset, view):
-        return self._filter_queryset(request, queryset, view, SearchQuerySet())
\ No newline at end of file
+        return self._filter_queryset(request, queryset, view, RelatedSearchQuerySet())
+
+
+class ConceptContainerPermissionedSearchFilter(HaystackSearchFilter):
+    def filter_queryset(self, request, queryset, view):
```

Comment:
There seems to be some incorrect behavior with with performing an `OR` on a queryset. Doing an or seems to return more results than are in either query sets. I suspect this is coming from `django-no-rel` but I cannot be sure until I investigate farther.